pkg/reconciler/autoscaling/kpa/kpa.go

```diff
@@ -131,7 +131,12 @@ func (c *Reconciler) ReconcileKind(ctx context.Context, pa *autoscalingv1alpha1.
 	//			this revision, e.g. after a restart) but PA status is inactive (it was
 	//			already scaled to 0).
 	// 2. The excess burst capacity is negative.
-	if want == 0 || decider.Status.ExcessBurstCapacity < 0 || want == scaleUnknown && pa.Status.IsInactive() {
+	if len(config.FromContext(ctx).Network.ActivatorCA) > 0 {
+		// When activator CA is enabled, force activator always in path.
+		// TODO: This is a temporary state and to be fixed.
+		// See also issues/11906 and issues/12797.
+		mode = nv1alpha1.SKSOperationModeProxy
+	} else if want == 0 || decider.Status.ExcessBurstCapacity < 0 || want == scaleUnknown && pa.Status.IsInactive() {
 		mode = nv1alpha1.SKSOperationModeProxy
```

Comment:
This almost feels like it should become something like a switch statement.

Also, you've moved line 134 to 139, but the comment on 127-133 relates directly to 139. Can we keep the comment next to the condition?

Reply:
Thank you! Updated.